tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/send_build_to_bugsnag.rb

```diff
@@ -172,72 +172,96 @@ def self.available_options
 
       private
 
-      def self.load_default_values
-        options = {releaseStage: "production", user: `whoami`.chomp}
+      # Get any Git options (remote repo, and revision) from the directory
+      def self.load_git_remote_options
```

Comment:
Moving methods around makes the diff larger than it needs to be and complicates the review - recommend making the smallest changeset possible in future!

Reply:
Acknowledged. The old order of the definitions didn't align after the changes made, but for future this would probably be best suited after the functional changes are completed and reviewed.